Sources/RuntimeAssertions/RuntimeAssertions.docc/RuntimeAssertions.md

```diff
@@ -0,0 +1,52 @@
+# ``RuntimeAssertions``
+
+<!--
+
+This source file is part of the Stanford XCTRuntimeAssertions open-source project
```

Comment:
We will need to update the READMe as well.

Reply:
That would be great; we can also rename the repo after that. Renaming it should still allow us to parse the older version of the Package, GitHub does redirects for repo renames: https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository